### PR TITLE
KBFS-3589 KBFS-3778 KBFS-3836: skip flakes

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: go/kbfs/kbfsgit/runner_test.go [hunk 1]
```diff
@@ -30,6 +30,7 @@ type testErrput struct {
 }
 
 func (te testErrput) Write(buf []byte) (int, error) {
+	te.t.Helper()
```
Comment (jakob223) [inline go/kbfs/kbfsgit/runner_test.go]: Why this change now?

Comment (jzila) [reply go/kbfs/kbfsgit/runner_test.go]: I was trying to comb through the logs and couldn't find the actual failing call because this one kept getting in the way.
